modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -224,138 +223,131 @@ xref:back-up-bonita-bpm-platform.adoc[Back up your platform] and database in cas
 | `bonita-migration-distrib-sp` (Linux) or `bonita-migration-distrib-sp.bat` (Windows)
 |===
 
- ** Starting from version 2.44.1, an additional script called `check-migration-dryrun` is present in the same folder. This script only run checks the migration would without actually migrating. This is equivalent to running the migration script with a `--verify` option.
-. The script detects the current version of Bonita, and displays a list of the versions that you can migrate to. Specify the
- version you require.
- The script starts the migration.
-. As the script runs, it displays messages indicating progress. After each migration step, you are asked to confirm whether to
- proceed to the next step. You can pause the migration by answering `no`.
- To suppress the confirmation questions, so that the migration can run unattended, set the ` (-Dauto.accept=true)` system
+ ** Starting from Bonita Migration Tool 2.44.1, an additional script called `check-migration-dryrun` is present in the same folder. This script only runs the checks the real update would run, without actually migrating the elements. This is equivalent to running the migration script with a `--verify` option.
+13. The script detects the current version of Bonita, and displays a list of the versions that you can update to. 
+14. Specify the version you require. 
+15. The updating procedure starts
+16. As the script runs, it displays messages indicating progress. After each step, you are asked to confirm whether to
+ proceed to the next step. You can pause the updating procedure by answering `no`. +
+ To suppress the confirmation questions, so that the update can run unattended, set the ` (-Dauto.accept=true)` system
  property.
- When the migration script is finished, a message is displayed showing the new platform version, and the time taken for the migration.
- The database have been migrated.
-
-WARNING: Do not use the old application server: a new one needs to be installed with the Bonita binaries that match the target version.
-
-. Unzip the target bundle version into a directory. In the steps below, this directory is called `bonita-target-version`.
-. xref:database-configuration.adoc[Configure the bundle to use the migrated database]. +
- Do not recreate the database and use the setup tool of the `bonita-target-version` Edit the `bonita-target-version/setup/database.properties` file to point to the  migrated database.
-. Reapply configuration made to the platform, using the setup tool of the `bonita-target-version`
-+
-Download the configuration from database to the local disk.
-+
-There is below a Linux example:
-+
+ 17. When the migration script is finished, a message is displayed showing the new Runtime version, and the time taken for the migration of all elements. The database has been updated.
```

Comment:
for the migration of all elements. => for migrating all the elements.

Reply:
fixed